DigitalLearningSolutions.Web/Controllers/Register/RegisterAdminController.cs

```diff
@@ -80,7 +98,17 @@ public IActionResult PersonalInformation(PersonalInformationViewModel model)
         {
             var data = TempData.Peek<RegistrationData>()!;
 
-            ValidateEmailAddress(model.PrimaryEmail, model.Centre!.Value);
+            if (!model.Centre.HasValue)
+            {
+                return new StatusCodeResult(500);
+            }

```

Comment:
What scenario in this post do we have a null centre id? I think in the GET we might just be able to navigate to the URL manually, but I'm struggling to see a way here.

Reply:
Right, I preserved this because it seemed like we should protect against a request getting sent with a null centreId, but if that happened it would error out anyway so we don't need these checks.